doc/user/raspi.rst

```diff
@@ -2,95 +2,41 @@
 Raspberry Pi
 ************
 
-:Last Reviewed: 2016-12-07
+:Last Reviewed: 2016-12-24
 
-This instructions were tested on Raspbian.
+This instructions were tested on Raspbian Jessie.
 
-.. warning:: You should **never** directly connect an printer with RS232-interface (serial port) directly to
```

Comment:
Is there a special reason that you have changed this warning box to a normal chapter?